File: app/src/main/java/com/fazziclay/opentoday/gui/interfaces/CurrentItemsTab.java
package com.fazziclay.opentoday.gui.interfaces;

import com.fazziclay.opentoday.app.items.tab.Tab;

import java.util.UUID;

public interface CurrentItemsTab {
    UUID getCurrentTabId();
    Tab getCurrentTab();
    void setCurrentTab(UUID id);
}

File: app/src/main/java/com/betterbrainmemory/opentoday/gui/interfaces/CurrentItemsTab.kt
package com.betterbrainmemory.opentoday.gui.interfaces

import java.util.UUID

interface CurrentItemsTab {
    fun getCurrentTabId(): UUID
    fun getCurrentTab(): com.betterbrainmemory.opentoday.app.items.tab.Tab
    fun setCurrentTab(id: UUID?)
}